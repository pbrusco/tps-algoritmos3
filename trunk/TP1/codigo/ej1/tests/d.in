958947	30503
64544	145429
326384	481721
901821	232459
888093	848083
434069	499091
297657	117839
708900	923095
378315	551147
738368	276641
987345	559201
556113	890015
258511	814949
920517	39813
580370	812829
472058	350079
763249	487501
164087	196401
87181	775755
747546	251245
901102	124861
703209	795811
546444	642607
749933	128743
324750	306099
709112	38293
897125	322259
198980	925793
974735	361277
267693	543483
128872	184695
348433	165557
458008	516683
961366	675641
711299	391
533749	838935
500014	714845
739911	913825
837289	48463
464745	733611
649440	708575
963445	814997
682934	292715
687387	853287
672510	395959
157952	453255
412324	872797
193165	546807
48070	707885
778075	172631
138216	338423
846790	309941
669770	962485
249816	494891
482849	683753
53316	347753
298072	965627
505705	267679
894303	161605
197826	7651
726027	338913
594693	440295
685407	803437
511699	850823
81721	99331
345712	564571
718785	572243
627919	510521
467096	88763
188477	664923
612764	914505
86985	914843
769768	198031
355136	158573
999578	250383
524438	850401
332103	623769
712464	413027
342553	925443
618522	163483
14037	851557
378832	464321
809688	379163
579456	250651
550983	449039
371707	686755
374480	366589
388517	386003
604579	564763
585251	480767
841271	810635
731416	392255
921907	623357
860128	10027
213489	973555
633163	553361
884546	844233
758500	211929
925346	860105
473744	340871
315015	588639
707270	761983
166437	837367
901690	26567
363744	115181
516472	349529
363771	312187
182585	896489
408731	829227
12031	513723
52848	49483
275704	735639
403200	177395
278556	766945
808926	311381
280203	619339
205817	501153
698469	805111
950409	567475
611978	742381
81685	24819
667464	427265
553202	181187
996464	602685
973242	732103
522237	150637
527010	838391
85737	231821
78362	61713
930289	903873
686640	245613
767265	811345
767849	338355
573382	910169
693097	253657
407292	116489
831720	360195
827582	563187
368706	485093
508559	574819
121396	557941
701393	199037
448525	231207
149642	303233
804589	576163
996329	496413
97936	420057
214424	81707
273990	958253
101572	699979
-1	-1
