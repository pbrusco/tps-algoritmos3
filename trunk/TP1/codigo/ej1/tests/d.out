3311
124768
326384
204444
149082
104760
61979
93025
437463
247833
428144
842222
258511
14109
249614
129026
105074
133334
73441
51216
80007
545839
605139
124358
205578
23932
81371
273186
47921
179037
151243
34119
377947
13750
162
451234
597319
55351
13418
595374
147225
366286
141594
710811
276551
417143
684787
468199
206330
99147
138216
194087
80840
214856
37728
321911
102185
238026
17488
1672
84321
414477
612001
511699
38615
77094
169990
246065
66133
151607
688199
86985
175675
37990
40697
80531
592722
299437
461710
128073
750598
101698
51362
7436
195941
110818
109436
239222
271600
66924
336996
14466
27146
9979
564664
447073
553982
193438
328231
255691
478428
707270
166437
6678
94288
166943
42281
52148
318158
346534
12143
459148
5105
689411
178758
537747
241886
698469
426024
611978
19768
222374
16831
138779
391913
80058
527010
85737
51551
88993
91845
697190
247589
341533
150596
36644
275010
42620
217060
564510
348524
104282
103882
126075
195400
270143
242020
51010
945968
230372
